MIGRATION.md

```diff
@@ -2,6 +2,14 @@
 
 ## Migrating from v10 to v11
 
+obt no longer supports bower, it now only supports npm.
```

Comment:
We should mention it only supports origami v2 and not origami v1 in this section

Reply:
I figure we can rewrite the whole migration guide to be more clear when it's done :) I'm seeing it as a list of changes for future reference at the moment